integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/MicroRegistryManager.java

```diff
@@ -119,6 +120,27 @@ public void updateResource(String resourcePath, String sourcePath, boolean backu
         }
     }
 
+    /**
+     * Function is to check if the registry resources are exist in filepath
+     *
+     * @param sourcePath path to registry resource
+     * @param fileName registry resource file name
+     */
+
+    public void checkResourceExist(String sourcePath, String fileName) throws MicroRegistryManagerException {
+
+        String path = sourcePath + fileName;
+
+        if (path != null ) {
+            log.info("Registry resources found");
```

Comment:
I think better we can return a Boolean and assert in the caller instead of throwing an exception. WDYT

Reply:
Added the requested change. Please review the latest commit.